### PR TITLE
ON-15072: Unify UBI versions used in Dockerfiles

### PR DESCRIPTION
**Pull request by tcrawley-xilinx**
## Testing done
Everything builds in container and accelerated traffic passes

### CODE REVIEW

**Review by pcolledg-xilinx — APPROVED** [review]

**Review by ivatet-amd — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -10,6 +10,29 @@ Expected environment:
   - Node Feature Discovery (NFD) Operator
   - Kernel Module Management (KMM) Operator ([Redhat docs](https://docs.openshift.com/container-platform/4.12/hardware_enablement/kmm-kernel-module-management.html), [codebase docs](https://kmm.sigs.k8s.io/documentation/deploy_kmod/))
 
+#### Base images required
+
+For onload & sfc:
+* `DTK_AUTO`
```
Comment (ivatet-amd) [inline README.md:16]: nit: Maybe list it as the last item (followed by the explanation of how to compute it)?

File: README.md [hunk 2]
```diff
@@ -10,6 +10,29 @@ Expected environment:
...
+* ubi8:8.8
+* ubi8-minimal:8.8
+* golang:1.20.4
+The DTK_AUTO image is determined by the KMM operator depending on the kernel
```
Comment (ivatet-amd) [inline README.md:20]:
```suggestion
The `DTK_AUTO` image is determined by the KMM Operator depending on the kernel
```

File: README.md [hunk 3]
```diff
@@ -10,6 +10,29 @@ Expected environment:
...
+versions of the nodes in your cluster. To get the location of this image you
+can run:
+```console
+$ oc adm release info $OCP_VERSION --image-for=driver-toolkit
+```
+Where `$OCP_VERSION` is the version of the openshift cluster on which you will
```
Comment (ivatet-amd) [inline README.md:26]:
```suggestion
Where `$OCP_VERSION` is the version of the OpenShift cluster on which you will
```

File: README.md [hunk 4]
```diff
@@ -10,6 +10,29 @@ Expected environment:
...
+deploy onload.
```
Comment (ivatet-amd) [inline README.md:27]:
```suggestion
deploy Onload.
```

**Review by ivatet-amd — APPROVED** [review]

**Pull request merged**